quack_app/test/golden_widget_test.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/widgets.dart';
+import 'package:quack_app/main.dart';
+import 'package:golden_toolkit/golden_toolkit.dart';
+
+void main() {
+
+  testWidgets('Golden test', (WidgetTester tester) async {
+    tester.binding.window.physicalSizeTestValue = const Size(1170, 2532);
+    tester.binding.window.devicePixelRatioTestValue = 3.0;
+    await loadAppFonts();
+    await tester.pumpWidget(const MyApp());
+    await expectLater(find.byType(MyApp),
+        matchesGoldenFile('main.png'));
```

Comment:
Can you update this to put all golden files in the following directory: `goldens/name_of_test.png` where `name_of_test` is more descriptive. Don't forget to delete `main.png`

Reply:
Addressed